app/javascript/components/widget/components/widget-pie-chart-legend/component.jsx

```diff
@@ -4,40 +4,57 @@ import { format } from 'd3-format';
 
 import PieChart from 'components/charts/pie-chart';
 import PieChartLegend from 'components/charts/components/pie-chart-legend';
+import Button from 'components/ui/button';
 
 import './styles';
 
 class WidgetPieChart extends PureComponent {
   render() {
-    const { data, settings, simple } = this.props;
+    const { data, settings, simple, toggleSettingsMenu } = this.props;
+    const noIntersectionSelected =
+      !settings.forestType && !settings.landCategory;
 
     return (
       <div className="c-pie-chart-legend-widget">
-        <PieChartLegend
-          className="cover-legend"
-          data={data}
-          config={{
-            format: '.3s',
-            unit: 'ha',
-            key: 'value',
-            ...settings
-          }}
-          simple={simple}
-        />
-        <PieChart
-          className="cover-pie-chart"
-          data={data}
-          maxSize={140}
-          tooltip={[
-            {
-              key: 'percentage',
-              unit: '%',
-              labelKey: 'label',
-              unitFormat: value => format('.1f')(value)
-            }
-          ]}
-          simple={simple}
-        />
+        {settings &&
+          settings.showSettingsBtn &&
+          noIntersectionSelected &&
+          toggleSettingsMenu && (
+          <Button
+            theme="theme-button-small theme-button-light"
+            className="pie-contextual-settings-btn"
+            onClick={() => toggleSettingsMenu()}
+          >
+              + Select an intersection
```

Comment:
this isnt generic, a widget might not have intersections but want this button to exist. It should be configurable.